site/cgi-bin/docker_ui.cgi

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import cgi
+import json
+import docker
+
+
+print("Content-type: text/html")
+print()
+
+
+def collectDockerInfo():
+    docker_info = {}
+    docker_images = {}
+
+    try:
+        docker_client = docker.from_env()
+        docker_info = docker_client.info()
+        docker_images_obj = docker_client.images.list()
```

Comment:
What of these three lines can throw an exception?

Reply:
`docker_client.info()` & `images.list()`l functions can throw `docker.errors.APIError` if the server returns an error.
https://docker-py.readthedocs.io/en/stable/client.html#docker.client.DockerClient.info
https://docker-py.readthedocs.io/en/stable/images.html#docker.models.images.ImageCollection.list